Backend/DataAccessLayer/Database/dbContext.cs

```diff
@@ -19,6 +19,24 @@ public DatabaseContext()
             this.Database.Connection.ConnectionString = connectionString;
         }
 
+        public void RevertDatabaseChanges(DatabaseContext _db)
```

Comment:
Added this function to undo database changes. Can be called wherever _db exists.